tests/libstorage/pvc.go

```diff
@@ -176,6 +176,15 @@ func CreateFSPVC(name, namespace, size string, labels map[string]string) *k8sv1.
 	return createPVC(pvc, namespace)
 }
 
+func CreateRWXFSPVC(name, namespace, size string) *k8sv1.PersistentVolumeClaim {
+	sc, _ := GetRWXFileSystemStorageClass()
```

Comment:
How do we handle a case when the storage class is missing?

Reply:
We skip the test in such case. We wanted to add a proper decorator, but unfortunately we don't have any yet. See: https://github.com/kubevirt/kubevirt/pull/13756#discussion_r1934103001